src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -1,7 +1,175 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { useFormik } from "formik";
+import { Measure } from "@madie/madie-models";
+import {
+  measureStore,
+  checkUserCanEdit,
+  routeHandlerStore,
+} from "@madie/madie-util";
+import MetaDataWrapper from "../../../editMeasure/details/MetaDataWrapper";
+import { QDMMeasureSchemaValidator } from "../../../../validations/QDMMeasureSchemaValidator";
```

Comment:
Looks like we don't need any validations

Reply:
Oops forgot to remove this. Done.